_posts/2020-04-07-hhvm-4.52.markdown

```diff
@@ -0,0 +1,28 @@
+---
+title: "HHVM 4.52"
+layout: post
+author: fred
+category: blog
+---
+
+HHVM 4.52 is released! This release marks the end of support for 4.46;
+HHVM 4.47&ndash;4.51 remain supported, as do the 4.8 and 4.32 LTS releases.
+
+# Highlights
+
+- fixed crash in `ext_soap` if a SoapServer's handler class does not exist.
+- `?>` and `<?=` are no longer recognized by the parser; this has replaced
+  dedicated errors with parse errors.
+
+# Breaking Changes
+
+- `__halt_compiler()` has been removed.
```

Comment:
I think it's been removed months ago (I see it announced in https://hhvm.com/blog/2019/12/10/hhvm-4.35.html), the current change might just be that it's no longer recognized by the parser so no longer gives a dedicated error (will just be "undefined function" now)?

Reply:
Yep, that's correct.